### DISCUSSION TITLE
Support for ARM64

### DISCUSSION THREAD

**Discussion by tranlm** [microsoft/onnxruntime-genai#508] [Support for Targets (OS / EPs / Hardware)]
Hi developers. I see in the readme that support for ARM64 is in progress. However, I don't see a branch or PR directly related to it. I'm wondering if there's a timeline or person I can speak with about development for this architecture. I'm really curious about (i) when the support will be enabled, and (ii) if I can get access to a development or experimental version for testing. Thanks all!

**Comment by natke** [comment]
Hi @tranlm, which O/S are you looking for Arm64 support on?

**Comment by IanET** [comment]
Any plans for Linux-Arm64 support?
